exporters/powerdns/powerdns-config.yml.sample

```diff
@@ -9,3 +9,12 @@ integrations:
 
         # API key used to connect to the PowerDNS server
         # api_key: ****-***-*****
+
+        transformations:
```

Comment:
As requested by @mangulonr  this metrics should be sent by default so we could add this configuration in a form of a comment with some explanatory message.

Reply:
make sense, I did not know it